lnwallet/config.go

```diff
@@ -41,6 +41,11 @@ type Config struct {
 	// update the commitment state.
 	Signer input.Signer
 
+	// The wallet's current ChannelContextSigner implementation.
+	// NOTE: This signer will subsume the input.Signer above when
```

Comment:
probably want a TODO here or on `Signer` above to make it clear that `Signer` *will* go away

Reply:
Done